src/DIRAC/TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -387,6 +395,11 @@ def cleanContent(self, directory):
         # Executing with shifter proxy
         gConfigurationData.setOptionInCFG("/DIRAC/Security/UseServerCertificate", "false")
         failed = {}
+        if self.cleanWithRMS:
```

Comment:
So actually this branch is never used for me, because everything is based on meta data to find files and not on directories